src/Templates/NIPSWeb/main.twig

```diff
@@ -7,7 +7,9 @@
 <div id="notice" class="alert alert-warning">Preparing show plan...</div>
 {# <div id="head-icons"><button class="btn-feedback">Send Feedback</button>&nbsp;<button class="btn-help"></button><button class="btn-logout" onclick="window.location = '/portal/redir_out.php'"></button></div> #}
 <header id="content-header">
-  <h3 id="a-manage-library"><a href="{{MyRadio.makeURL(config,'NIPSWeb','manage_library')}}" title="Library Manager">Manage Library</a></h3>
+  <div id="baps-menu" class="btn-group">
+    <a  href="{{MyRadio.makeURL(config,'NIPSWeb','manage_library')}}" id="menu-track-upload" class="btn btn-default"><span class="glyphicon glyphicon-upload"></span> Upload Tracks</a>
```

Comment:
Not sure on the change of phrasing here - to me "Tracks" are things in the central library, not things like beds/jingles, which you can also manage here.